src/views/Task/TaskView.css

```diff
@@ -0,0 +1,10 @@
+.view {
+  height: 100vh;
+  padding : 5%;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
Try doing the above-mentioned section tag, don't think you need the component-specific CSS 👍